ts/accountTree.ts

```diff
@@ -25,34 +27,78 @@ export class AccountRegistry {
     ) {
         this.treeLeft = Tree.new(depth);
         this.treeRight = Tree.new(depth);
-        this.setSize = 1 << depth;
+        this.setSize = 2 ** depth;
```

Comment:
JS bitwise operation is signed. Power operation is a way to get correct 31 depth value
Number.MAX_SAFE_INTEGER is `2**53 -1`, so we are still safe in Number.

```
$ node
Welcome to Node.js v12.14.1.
Type ".help" for more information.
> 1 << 31
-2147483648
> 2 ** 31
2147483648
> (1 << 31) >>> 0
2147483648
```